kong/db/dao/plugins/go.lua

```diff
@@ -62,6 +63,20 @@ do
     return out:match("Runtime Version: go(.+)\n$")
   end
 
+  local function grab_logs(proc)
+    while true do
+      local data, err, partial = proc:stdout_read_line()
+      local line = data or partial
+      if line and line ~= "" then
+        errlog.raw_log(ngx.INFO, "[go-pluginserver] " .. line)
```

Comment:
As this could potentially be a hot path, maybe it makes sense to cache `errlog.raw_log` and `ngx.INFO` at module level.